packages/inventory-compliance/src/Utilities/RulesHelper.js

```diff
@@ -1,15 +1,14 @@
-export const toRulesArray = (policiesWithRules) => {
-    const rules = policiesWithRules.flatMap((policy) => (
-        policy.rules.map((rule) => {
+export const toRulesArray = (profilesAndRules) => (
+    profilesAndRules.flatMap((record) => (
```

Comment:
Please use either `profile` or `policy` which ever one is more appropriate.

Reply:
Neither is in this case. `record` changed to `profileRule`, and `profilesAndRules` to `profileRules`.